command_test.go

```diff
@@ -133,3 +134,22 @@ func Test_DisableFlagParsing(t *testing.T) {
 		t.Errorf("expected: %v, got: %v", as, targs)
 	}
 }
+
+func TestCommandsAreSorted(t *testing.T) {
+
+	originalNames := []string{"middle", "zlast", "afirst"}
```

Comment:
I would prefer you create a manual list of output names, so we aren't using the `sort` package to test the `sort` package...